python/pyspark/ml/tests/test_feature.py

```diff
@@ -989,148 +990,22 @@ def test_target_encoder_binary(self):
             targetType="binary",
         )
         model = encoder.fit(df)
-        te = model.transform(df)
```

Comment:
the old tests are actually identical with scala side, so I simplify it to mainly focus on save/load